index.html

```diff
@@ -21,52 +22,42 @@ <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
         <div>
-            <ul>
-                <li>Phillip Cortez Temple</li>
-                <li>
-                    <img src="https://avatars2.githubusercontent.com/u/44590915?s=460&v=4" alt="Pic of Phill">
-                </li>
-                <li>
-                    We are all born ignorant, but one must work hard to remain stupid. Benjamin Franklin
-                </li>
-                <li>
-                    phillipcorteztemple@gmail.com
-                </li>
-            </ul>
+            <div id="phill">
+                <div id="phill ul">
```

Comment:
This `div` has two `ids`: `phill` and `ul`... neither of which makes much sense. Remember, attributes like `id` and `class` are space-separated lists!

This is _not_ to be confused with a complex selector on the CSS side like `#phill ul`. This 👈 selector is selecting all of the indirect `ul` children of an element with an `id` of `phill`. It is _not_ a unique id `phill ul`.

Reply:
I think is it is fixed.  I am going to review the notes.